api/tests/unit/domain/events/handle-certification-rescoring_test.js

```diff
@@ -7,12 +7,119 @@ import { CertificationJuryDone } from '../../../../lib/domain/events/Certificati
 import { CertificationAssessment, CertificationResult, AssessmentResult } from '../../../../lib/domain/models/index.js';
 import { CertificationComputeError } from '../../../../lib/domain/errors.js';
 import { CertificationVersion } from '../../../../src/shared/domain/models/CertificationVersion.js';
+import { config } from '../../../../src/shared/config.js';
+import _ from 'lodash';
 
 const CERTIFICATION_RESULT_EMITTER_AUTOJURY = CertificationResult.emitters.PIX_ALGO_AUTO_JURY;
 const CERTIFICATION_RESULT_EMITTER_NEUTRALIZATION = CertificationResult.emitters.PIX_ALGO_NEUTRALIZATION;
 
 describe('Unit | Domain | Events | handle-certification-rescoring', function () {
   describe('when handling a v3 certification', function () {
+    describe('when less than minimumAnswersRequiredToValidateACertification have been answered', function () {
```

Comment:
when less than the minimum number of answers required by the config has been answered